tests/phpunit/WC_Stripe_Helper_Test.php

```
@@ -279,6 +279,97 @@ public function provide_test_get_stripe_amount(): array {
+		];
+	}
+
+	/**
+	 * Test for `get_woocommerce_amount_from_stripe_amount` with empty currency (uses store currency).
+	 */
+	public function test_get_woocommerce_amount_from_stripe_amount_falls_back_to_store_currency(): void {
+		update_option( 'woocommerce_currency', 'EUR' );
+		$result = WC_Stripe_Helper::get_woocommerce_amount_from_stripe_amount( 19999, '' );
+		$this->assertIsString( $result );
+		$this->assertSame( '199.99', $result );
+	}
```

PHPMD (2.15.0)

[error] The method test_get_woocommerce_amount_from_stripe_amount_falls_back_to_store_currency is not named in camelCase. (undefined)

(CamelCaseMethodName)